.github/workflows/ci.yml

```diff
@@ -36,3 +36,35 @@ jobs:
 
       - name: Run tests
         run: npm test
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
Does this mean every push to master updates Docker? Shouldn't that be limited to releases of the package (i.e., to npm)?

Reply:
So I went with each push to master so that if someone wants to use what is in the master branch, it's immediately available and it isn't put on the user to build this from scratch. Once this is in place, we could then add a release workflow that would also create a tagged version of the image.

Reply:
I think I'd prefer doing releases first - or at least together. I'd also prefer a trigger, I think - most of the commits are Dependabot and there's not much point publishing a new release because a devDependency updated.

Reply:
That makes sense. I can make some changes to make it happen that way and then if we see that there is a need to make more frequent releases we can always add it back later.